doc/reference/introduction.md

```diff
@@ -0,0 +1,351 @@
+---
```

Comment:
Would it be better for blame if this new file was first moved from introduction.md and then changed? Or maybe the blame tracker doesn't go across renamed files....  not a blocker -- just wondered...